internal/pkg/template/templates/environment/partials/cdn-resources.yml

```
@@ -0,0 +1,24 @@
+cloudfrontdistribution:
+  Metadata:
+    'aws:copilot:description': 'A CloudFront distribution for global content delivery'
+  Condition: CreateALB
+  Type: AWS::CloudFront::Distribution
+  Properties:
+    DistributionConfig:
+      DefaultCacheBehavior:
+        AllowedMethods: ["GET", "HEAD", "OPTIONS", "PUT", "PATCH", "POST", "DELETE"]
+        CachePolicyId: 4135ea2d-6df8-44a3-9df3-4b5a84be39ad
```

😮 How is this value determined?

---

The cache policy id is just a global id given by AWS, so I'm fairly certain can be hardcoded in this code.
[Here](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-cache-policies.html) is more about this.

---

The cache policy id is just a global id given by AWS, so I'm fairly certain can be hardcoded in this code.
[Here](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-cache-policies.html) is more about this.

---

Nice! can you add this as a comment next to the value?
```suggestion
        CachePolicyId: 4135ea2d-6df8-44a3-9df3-4b5a84be39ad # See https://go.aws/3bJid3k
```